www/map-view-controller.js

```diff
@@ -358,7 +358,31 @@ class MapViewControllerImpl {
    * @method
    */
   setLanguage(language) {
-    this._sendMessageToViewer('ui.set_language', language)
+    this._sendMessageToViewer('ui.set_language', language);
+  }
+
+  /**
+   * @typedef {Object} MapViewDirectionsOptions - Options to apply to directions requests
+   * @property {string[]} includedTags - List of tags that you want to use when calculating a route. Only the tags added here will be used. If there are other tags in the graph they won't be used. The edges without a tag will be used. If you don't set nothing all the graph will be used to calculate the route. You can learn more about this topic on https://situm.com/docs/cartography-management/#tags
```

Comment:
If you don't set nothing **at** all

Reply:
Que pasa si seteas includedTags e excludedTags ?
En teoria creo que se priorizaban os included. Hay que explicarlle ao integrador que debe usar un dos dous solo.